src/utils/exif/index.ts

```diff
@@ -0,0 +1,25 @@
+export function prettyPrintExif(data: Object) {
+    let a;
+
+    let strPretty = '';
+    for (a in data) {
+        if (typeof data[a] === 'object') {
+            if (data[a] instanceof Number) {
+                strPretty +=
+                    a +
+                    ' : ' +
+                    data[a] +
+                    ' [' +
+                    data[a].numerator +
+                    '/' +
+                    data[a].denominator +
+                    ']\r\n';
+            } else {
+                strPretty += a + ' : [' + data[a].length + ' values]\r\n';
+            }
+        } else {
```

Comment:
When the type isn't of `object` type, then it will be always of type `string`?

Reply:
any thing other than object can be printed as it is